tinywallet/tinywallet/config.py

```diff
@@ -102,6 +102,48 @@ class DB:
     wallet = "wallet".encode()
     theme = "theme".encode()
     dcrdata = "dcrdata".encode()
+    rpcuser = "rpcuser".encode()
+    rpcpass = "rpcpass".encode()
+    rpccert = "rpccert".encode()
+    rpchost = "rpchost".encode()
+    dcrdon = "dcrdon".encode()
+
+
+DcrdPorts = {
+    nets.mainnet.Name: "9109",
+    nets.testnet.Name: "19109",
+    nets.simnet.Name: "19556",
+}
```

Comment:
Shouldn't these port numbers live in the `dcrd.nets` package, together with the rest of the network parameters?